packages/react/src/components/LoaderOverlay/index.tsx

```diff
@@ -7,37 +7,65 @@ import AxeLoader from './axe-loader';
 interface LoaderOverlayProps extends React.HTMLAttributes<HTMLDivElement> {
   variant?: 'large' | 'small';
   label?: string;
+  focus?: boolean;
+  children?: React.ReactNode;
```

Comment:
Added props for focus and children. Typescript does not seem to like leaving `children` as usable under `...others` without it having a type definition.